tools/run_cali.sh

```diff
@@ -2,20 +2,21 @@
 selected_data=cafe1-1
 num_camera=2
 cam_index=0
+bin_path='../../../build/CAMODOCAL/bin'
 
-if [[ $1 -eq "" ]]
+if [ $1 == "" ]
 then
     echo "Missing path to input data !"
     exit
 fi
 echo "************************************************"
 echo "Input data ${selected_data}, sanity checking ..."
 echo "************************************************"
-python3 checker.py ${num_camera} $1/data/${selected_data}/camodocal_input_data/
+python3 checker.py ${num_camera} $1/camodocal_input_data/
 echo "Press any key to continue ..."
 read -n 1
 
-extrinsic_calib -c $1/data/${selected_data}/camodocal_input_caminfo --camera-count ${num_camera} --camera-index ${cam_index} -m 100 --input $1/data/${selected_data}/camodocal_input_data/ --keydist 0.1 -v 
```

Comment:
about the selected_data, actually you don't need to change that too often , having this inside as an variable, save me from type the selected_data every time when I launch the run_calib.sh.   (I just run ./run_calib.sh ~)